users/management/commands/retire_users.py

```diff
@@ -54,44 +59,42 @@ def add_arguments(self, parser):
             action="append",
             default=[],
             dest="users",
-            help="Single or multiple username(s)",
+            help="Single or multiple username(s) or email(s)",
         ),
 
     def get_retired_email(self, email):
         """ Convert user email to retired email format. """
         return user_util.get_retired_email(email, RETIRED_USER_SALTS, RETIRED_EMAIL_FMT)
 
     def handle(self, *args, **kwargs):
-        usernames = kwargs.get("users", [])
+        users = kwargs.get("users", [])
 
-        if not usernames:
+        if not users:
             self.stderr.write(
                 self.style.ERROR(
-                    "No username(s) provided. Please provide username(s) using -u or --user."
+                    "No user(s) provided. Please provide user(s) using -u or --user."
                 )
             )
             exit(1)
 
-        for username in usernames:
-            self.stdout.write("Retiring user: {username}".format(username=username))
+        for u in users:
+            self.stdout.write("Retiring user: {user}".format(user=u))
```

Comment:
`user`

Reply:
I deliberately changed this because `user` is already being occupied inside the loop.

Reply:
So then please rename that inside loop user to existing_user